packages/@lwc/module-resolver/src/utils.ts

```diff
@@ -54,31 +98,104 @@ export function getModuleEntry(moduleDir, moduleName) {
 export function normalizeConfig(config: Partial<ModuleResolverConfig>): ModuleResolverConfig {
```

Comment:
Dont follow this one

Reply:
See comments on error handling above. I think we need to treat especially `getModuleEntry` in the case where we can't find the entry point instead of silencing the resolution error.